backend/src/test/resources/queries/person-with-last-test-result-query

```diff
@@ -5,7 +5,7 @@ query getPatientsWithLastTestResult($namePrefixMatch: String = null) {
     birthDate
     telephone
     lastTest {
-      dateAdded
+      createdAt
```

Comment:
Just to minimize churn, could we leave this as `dateAdded`?

Reply:
Sure, good call. I'll revert the rename.

Do you know if there's a reason we need to use the date from the TestOrder rather than the TestEvent? That extra step is one of the query layers, found in `TestResultDataResolver.getDateAdded`

Reply:
Looking deeper, I think that `TestEvent.getDateTested` is the most accurate representation of the _intent_ of this field, given that it will use the backdate if one is present. But, lacking the history, I'm still hesitant to make this change